session.go

```diff
@@ -398,5 +398,11 @@ func CreateSession(store kv.Storage) (Session, error) {
 
 	variable.BindSessionVars(s)
 	variable.GetSessionVars(s).SetStatusFlag(mysql.ServerStatusAutocommit, true)
+	b, ok := storeBootstrapped[store.UUID()]
```

Comment:
I will check if system db exists in bootstrap(). If system db already exists, I will do nothing.